recipes/simdjson/0.4.6/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class SimdjsonConan(ConanFile):
+    name = "simdjson"
+    version = '0.4.6'
+    description = "Parsing gigabytes of JSON per second"
+    topics = ("conan", "json", "parser", "simd", "format")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lemire/simdjson"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "threads": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True,
+                       'threads': True}
+    _source_subfolder = "source_subfolder"
+
+    @property
+    def _supported_cppstd(self):
+        return ["11", "gnu11", "17", "gnu17", "20", "gnu20"]
+
+    def _has_support_for_cpp11(self):
+        supported_compilers = [("apple-clang", 10), ("clang", 6), ("gcc", 7), ("Visual Studio", 15.7)]
+        compiler, version = self.settings.compiler, Version(self.settings.compiler.version)
+        return any(compiler == sc[0] and version >= sc[1] for sc in supported_compilers)
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            self.options.remove("fPIC")
```

Comment:
```suggestion
            del self.options.fPIC
```

Reply:
Done.

Reply:
I didn't explain, but `self.options.remove("fPIC")` has been deprecated. Once approved in EAP, the CCI bot will check your recipe and decline.

del is more pythonic, just like delete in C++, so we don't re-implement the wheel

Reply:
That's fine. I am fluent in Python and I know about del.

Please keep the comments coming!